### PR TITLE
[DDP-8198] update facebook and twitter links

### PR DESCRIPTION
**Pull request by LKapanad**

### CODE REVIEW

File: pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/housekeeping/PubSubMessageBuilder.java [hunk 1]
```diff
@@ -291,20 +284,7 @@ public PubsubMessage createMessage(String ddpMessageId,
             throw new MessageBuilderException("message has not been determined for queued event " + pendingEvent
                     .getQueuedEventId());
         }
-        PubsubMessage.Builder messageBuilder = PubsubMessage.newBuilder()
-                .setData(ByteString.copyFromUtf8(messageJson))
-                .putAttributes(DDP_EVENT_ID, pendingEvent.getDdpEventId())
-                .putAttributes(DDP_EVENT_TYPE, pendingEvent.getActionType().name())
-                .putAttributes(DDP_MESSAGE_ID, ddpMessageId)
-                .putAttributes(DDP_EVENT_CONFIGURATION_ID, Long.toString(pendingEvent.getEventConfigurationId()))
-                .putAttributes(DDP_STUDY_GUID, studyGuid)
```
Comment (gskhirel) [inline pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/housekeeping/PubSubMessageBuilder.java]: Why did you remove this?

Comment (LKapanad) [reply pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/housekeeping/PubSubMessageBuilder.java]: It was not a part of this PR.

**Review by gskhirel — DISMISSED** [review]

**Review by gskhirel — APPROVED** [review]
